content/en/tracing/setup/dotnet.md

```diff
@@ -308,7 +304,7 @@ The .NET Tracer can instrument the following libraries automatically:
 
 Notes:
 
-<sup>1</sup> The `AspNet` integration adds instrumentation to any ASP.NET application based on `System.Web.HttpApplication`, which can include applications developed with Web Forms, MVC, Web API, and other web frameworks. To enable the `AspNet` integration, you must add the [`Datadog.Trace.ClrProfiler.Managed`][4] NuGet package to your application.
+<sup>1</sup> The `AspNet` integration adds instrumentation to any ASP.NET application based on `System.Web.HttpApplication`, which can include applications developed with Web Forms, MVC, Web API, and other web frameworks. To enable the `AspNet` integration, you must add the [`Datadog.Trace.ClrProfiler.Managed`][4] NuGet package to your application. Be sure to keep this package in sync with your MSI version.
```

Comment:
I'm all for consolidating the usage of the `Datadog.Trace.ClrProfiler.Managed`! My only concern is, do you think a user will find this before coming to support to ask why their AspNet integration is not working? I personally think this change this looks fine, but I'm wondering if there are differing viewpoints

Reply:
The basic setup flow for customers is now 1) Install the .NET tracer installation package based on OS, 2) Look at the integrations you want to use, 3) Notice that AspNet requires the NuGet package and only at that point add the NuGet package to your project

Reply:
I think it doesn't hurt to have here. We should definitely have a better flow for upgrades if they rely on it.